### PR TITLE
Clicking the current breakpoint should not deselect the breakpoint.

### PR DESCRIPTION
**Pull request by wasimTQ**
## Description

1. Prevents the breakpoint to be deselected #1764

## Steps for reproduction

1. Go to the builder screen
2. Change the breakpoint
3. Click on the changed breakpoint again

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [x] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 2e6d)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Comment by wasimTQ** [comment]
@kof @TrySound How can I move from here?

File: apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx
Comment (kof) [inline apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx:68]: according to types, it can't be empty, so something is wrong here

Comment (wasimTQ) [reply apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx:68]: Maybe we can use length?

Comment (kof) [reply apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx]: @TrySound please check this out, strange, it shouldn't be an empty string ever from what I know

Comment (wasimTQ) [reply apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx]: https://www.radix-ui.com/docs/primitives/components/toggle-group#item

You can see the toggles can be turned on and off. It sends a empty string once the toggle to the `onValueChange` function.

File: apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx
Comment (TrySound) [inline apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx:68]: I'd say the second check + comment is enough.

Comment (wasimTQ) [reply apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx:68]: > I'd say the second check + comment is enough.

Got it

File: apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx
Comment (wasimTQ) [reply apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx]: @kof If it looks good on the last commit. Can you merge it?

File: apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx
Comment (kof) [inline apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx:68]: we are not writing stuff like `!something` because when it comes to readability you may get `!i` which is super hard to read on many fonts

Comment (kof) [inline apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx:68]: Proper comment for this:
```suggestion
          // When user clicks on a selected breakpoint, we don't want to deselect it.
          if (breakpoints.has(breakpointId) === false) {
```

**Comment by TrySound** [comment]
Nope, something with chromatic and lost pixel. Try empty commit.

**Comment by wasimTQ** [comment]
> Nope, something with chromatic and lost pixel. Try empty commit.

@TrySound This workflow requires approval from a maintainer. Can you look into it?

**Comment by TrySound** [comment]
I guess you just don't have access to those. It's ok. Main workflow is the most important here.

**Comment by wasimTQ** [comment]
> I guess you just don't have access to those. It's ok. Main workflow is the most important here.

Now if I do empty commit will it pass?

**Comment by kof** [comment]
@TrySound you don't see the "approve and run" button from github? I see it and click when I see it. I thinks we need to figure out a way for people to contribute and run ci without us clicking it, unless its impossible, not sure what other companies do, we can look into how cal.com is handling this

**Comment by TrySound** [comment]
I saw it and clicked on it

**Review by TrySound — APPROVED** [review]

**Comment by TrySound** [comment]
Thank you!

**Pull request merged**
